packages/lit-dev-content/src/pages/playground.ts

```diff
@@ -143,29 +144,31 @@ window.addEventListener('DOMContentLoaded', () => {
   window.addEventListener(CODE_LANGUAGE_CHANGE, syncStateFromUrlHash);
 
   // Trigger URL sharing when Control-s or Command-s is pressed.
-  let controlDown = false;
-  let commandDown = false;
-  window.addEventListener('keydown', (event) => {
-    if (event.key === 'Control') {
-      controlDown = true;
-    } else if (event.key === 'Meta') {
-      commandDown = true;
-    } else if (event.key === 's' && (controlDown || commandDown)) {
-      share();
-      event.preventDefault(); // Don't trigger "Save page as"
-    }
-  });
-  window.addEventListener('keyup', (event) => {
-    if (event.key === 'Control') {
+  if (!MODS?.split(' ').includes('gists')) {
```

Comment:
No action required.
Nit: Worth exporting a helper function from `mods.js` that checks for a mod. `hasMod(<mod>)`?

Reply:
Done